app/experimenter/experiments/migrations/0090_platforms_to_array.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 3.0.4 on 2020-04-06 23:36
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+import experimenter.experiments.models
+
+
+def forward_platform_string_to_array(apps, schema_editor):
+    Experiment = apps.get_model("experiments", "Experiment")
+
+    for experiment in Experiment.objects.all():
+        experiment.platforms = [experiment.platform]
```

Comment:
What about the `All Platforms` case?  Shouldn't this add the list of all three?

Reply:
Yes, oops, this of course needs to be changed to address the "All Platforms" case.